### ISSUE TITLE
asciidoctor errors reported while building site locally

### ISSUE THREAD

**Issue by sandrobonazzola** [oVirt/ovirt-site#2960]
#### Type of problem?
Documentation issue

Local site builds shows:
```
asciidoctor: ERROR: assembly_incremental-backup-and-restore.adoc: line 63: level 0 sections can only be used when doctype is book
asciidoctor: ERROR: assembly_incremental-backup-and-restore.adoc: line 76: level 0 sections can only be used when doctype is book
asciidoctor: ERROR: assembly_incremental-backup-and-restore.adoc: line 88: level 0 sections can only be used when doctype is book
asciidoctor: ERROR: index.adoc: line 20: invalid part, must have at least one section (e.g., chapter, appendix, etc.)
asciidoctor: ERROR: proc-Applying_the_DISA_STIG_Security_Profile_RHV.adoc: line 40: level 0 sections can only be used when doctype is book
asciidoctor: ERROR: proc-Applying_the_PCI_DSS_Security_Profile_RHV.adoc: line 30: level 0 sections can only be used when doctype is book
asciidoctor: WARNING: proc-Preparing_NFS_Storage2.adoc: line 79: unterminated listing block
```

**Comment by apinnick** [comment]
I can fix the incremental-back-and-restore problem. It looks like the first header was commented out.

However, I do not think I can fix the security profile errors. I suspect that they are caused by the fact that these proc files are only used in the downstream build.

**Comment by sandrobonazzola** [comment]
Fixed by #2961
